src/test/java/step1/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package step1;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 size() 메서드로 정확한 크기를 가져오는데 성공하는 케이스")
+    public void dateTypeSetSizeTest() {
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
컬렉션 사이즈를 검증하는 메소드를 사용해보면 어떨까요?
```suggestion
        assertThat(numbers).hasSize(3);
```